src/test/java/fr/lunatech/timekeeper/services/responses/UserEventResponseTest.java

```diff
@@ -13,34 +14,43 @@
     @Test
     void shouldCreateFromUserEvent() {
         var userEvent = new UserEvent();
+        var user = new User();
+        user.id = 1L;
         userEvent.id = 1L;
         userEvent.name = "Event name";
         userEvent.description = "Event description";
+        userEvent.owner = user;
         var response = UserEventResponse.createFromUserEvent(userEvent);
-
+        //TODO GUILLAUME CEST QUOI ÇA'??????????????????????????
```

Comment:
Please, no personal comment in the code (remember : the source code IS public and open-source)